src/functions-and-arrays.js

```diff
@@ -1,86 +1,200 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 >= num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
 
+// Iteration #2: Find longest word
+const words = ["mystery", "brother", "aviator", "crocodile", "pearl", "orchard", "crackpot"];
 
+function findLongestWord(words) {
+  if (words.length === 0) {
+    return null;
+  }
 
-// Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+  if (words.length === 1) {
+    return words[0];
+  }
 
-function findLongestWord() {}
+  let wordLengths = words.map((word) => {
```

Comment:
I would be more efficient to store the word in a string to avoid having multiple loops